### PR TITLE
Feature/dx 118 resources

### PR DESCRIPTION
**Pull request by trieutran91**
I've created a resources section according to DX-118.
It needs more content later, but it is ready to be applied to the master branch.

### CODE REVIEW

File: resources/demoshop.md [hunk 1]
```diff
@@ -0,0 +1,35 @@
+---
+title: Swedbank Pay Resources – Introduction
+opengraph:
+    description: Introduction to Swedbank Pay Resources
+sidebar:
+  navigation:
+  - title: Checkout
```
Comment (asbjornu) [inline resources/demoshop.md]: I don't think Checkout's menu should be on the `resource` pages. :)

File: resources/index.md [hunk 2]
```diff
@@ -0,0 +1,30 @@
+---
+title: Swedbank Pay Developer Portal – Resources
+opengraph:
+    description: Swedbank Pay Developer Resources
+sidebar:
+  navigation:
+  - title: Checkout
```
Comment (asbjornu) [inline resources/index.md]: I don't think Checkout's menu should be on the `resource` pages. :)

File: resources/test-data.md [hunk 3]
```diff
@@ -0,0 +1,240 @@
+---
+title: Swedbank Pay Resources – Introduction
+opengraph:
+    description: Introduction to Swedbank Pay Resources
+sidebar:
+  navigation:
+  - title: Checkout
```
Comment (asbjornu) [inline resources/test-data.md]: I don't think Checkout's menu should be on the `resource` pages. :)

File: resources/demoshop.md (hunk 1 above)
Comment (trieutran91) [reply resources/demoshop.md]: Agree! It's unnecessary to have that menu in the resources section. Removed

File: resources/index.md (hunk 2 above)
Comment (trieutran91) [reply resources/index.md]: Agree! It's unnecessary to have that menu in the resources section. Removed

File: resources/test-data.md (hunk 3 above)
Comment (trieutran91) [reply resources/test-data.md]: Agree! It's unnecessary to have that menu in the resources section. Removed

**Pull request merged**